modules/ebi-metagenomics/summarisegoslims/main.nf

```diff
@@ -15,26 +15,24 @@ process SUMMARISEGOSLIMS {
     path go_banding
 
     output:
-    tuple val(meta), path("*_summary.csv"), emit: go_summary
-    tuple val(meta), path("*_slim.csv")   , emit: goslim_summary
-    path "versions.yml"                   , emit: versions
+    tuple val(meta), path("*_summary.tsv"), emit: go_summary
+    tuple val(meta), path("*_slim.tsv"),    emit: goslim_summary
+    path "versions.yml",                    emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    summarise_goslims \\
+    # This is a modified version that generates a TSV which is not all quoted and it's sorted
```

Comment:
maybe write TODO in this comment so we can easily remember to make changes to the toolkit

Reply:
I will submit a PR for that in a bit :)